lib/src/test/java/vmath/Mat2Test.java

```diff
@@ -9,6 +9,21 @@ public class Mat2Test {
 
     @Test
     void givenNoArgs_whenMat2Created_thenMatrixIsIdentity() {
-        //TODO
+        Mat2 matrix = new Mat2();
+
+        assertEquals(1.0f, matrix.get(0, 0));
+        assertEquals(0.0f, matrix.get(0, 1));
+        assertEquals(0.0f, matrix.get(1, 0));
+        assertEquals(1.0f, matrix.get(1, 1));
+    }
+
+    @Test
+    void givenIdentityMatrix_whenGetElement_thenReturnValueMatches() {
+        Mat2 matrix = new Mat2();
+
+        assertEquals(1.0f, matrix.get(0, 0));
+        assertEquals(0.0f, matrix.get(0, 1));
+        assertEquals(0.0f, matrix.get(1, 0));
+        assertEquals(1.0f, matrix.get(1, 1));
     }
```

Comment:
Create a test to check for invalid indices. Initialize an empty test method with a TODO comment to test the `get` method. We will wait to implement this test until more complex constructors are available. See @vgdaut's code for reference [here](https://github.com/toksaitov/vmath/pull/46/files).